### ISSUE TITLE
Module doesn’t export Datastar

### ISSUE THREAD

**Issue by bencroker** [starfederation/datastar#358]
Getting an error when trying to import Datastar (core and full) into a module.

```js
import {Datastar} from 'https://cdn.jsdelivr.net/gh/starfederation/datastar/bundles/datastar.js';
```

Results in:

```
Uncaught SyntaxError: The requested module 'https://cdn.jsdelivr.net/gh/starfederation/datastar/bundles/datastar.js' doesn't provide an export named: 'Datastar'
```

**Comment by delaneyj** [comment]
Fixed with #359

**Comment by bencroker** [comment]
Thanks!
